### PR TITLE
Replace dig with whatsmydns.net

### PR DESCRIPTION
**Pull request by ari-gold**
Based on feedback in two separate doc status reports that users on Windows couldn't use dig

## Remaining Work
- [x] Copy review

### CODE REVIEW

**Comment by alexfornuto** [comment]
@ari-gold I'm not comfortable with our documentation depending on the availability and possibly changing status of an external resource like that. I think the better solution, which I've put into #3462 would be to either:

A) Offer a link to the Dig for Windows binary
B) Change the docs to use `nslookup` which is pretty universal to all OSes, including Windows. 

Right now I'm leaning towards B. Thoughts?

**Comment by ari-gold** [comment]
@alexfornuto  I think for someone not comfortable with the command line that an easy to use web tool is the best option and that installing dig or using nslookup introduces a lot of unnecessary friction to an already multi-step process of going live. Yes, it's a dependency, but not one I'm concerned about, and if it becomes a problem, we'll always be able to update the one or two lines with another easy to use web tool.

**Comment by alexfornuto** [comment]
> installing dig or using nslookup

nslookup is built in to Windows.

>  if it becomes a problem

I'd prefer it never became a problem.

File: source/_docs/domains.md [hunk 1]
```diff
@@ -195,21 +195,15 @@ Replace `http://` in the site's database and configure your CMS to assume users
 ### CNAME Record Workaround
 If your domain's DNS configuration relies on an existing MX or TXT record that intentionally disallows CNAME records, you'll need to use A and AAAA records to configure DNS for subdomains (e.g., `www.example.com`) instead of CNAMEs:
 
-1. Identify required A record value by querying the target environment's [platform domain](#platform-domains) (e.g., `live-example.pantheonsite.io`) from the command line using dig:
+1. Identify required A record value by querying the target environment's [platform domain](#platform-domains) (e.g., `live-example.pantheonsite.io`) using a free online tool: [https://www.whatsmydns.net/#A/{live-example.pantheonsite.io}](https://www.whatsmydns.net/#A/{live-example.pantheonsite.io}).
```
Comment (rachelwhitton) [inline source/_docs/domains.md]: Suggest removing `{` and `}` from the URL to reduce confusion around how to structure the URL. `live-example.pantheonsite.io` should be enough to signal example platform domain.

**Comment by rachelwhitton** [comment]
fwiw I think this makes sense based on feedback received, but I could be convinced that dig or nslookup is needed for those who prefer to work from the command. Not saying we need to do one or the other, but for the web tool does feel more accessible.

**Comment by alexfornuto** [comment]
@rachelwhitton edited based on feedback. Feel free to merge.

**Pull request merged**